package.json

```diff
@@ -87,7 +87,8 @@
     "webpack": "^5.61.0",
     "webpack-bundle-tracker": "^1.2.0",
     "webpack-cli": "^5.0.1",
-    "winchan": "^0.2.1"
+    "winchan": "^0.2.1",
+    "zod": "^3.20.6"
```

Comment:
We also want to bump [PROVISION_VERSION](https://zulip.readthedocs.io/en/latest/subsystems/dependencies.html#provision-version) by a minor level since we add a dependency here.